model_cards/german-nlp-group/electra-base-german-uncased/README.md

```diff
@@ -0,0 +1,166 @@
+---
+language: de
+license: mit
+thumbnail: "https://raw.githubusercontent.com/German-NLP-Group/german-transformer-training/master/model_cards/german-electra-logo.png"
+tags:
+- electra
+- german
+- deutsch
+- commoncrawl
+- base
+- uncased
+- umlaute
+- BERT
+---
+
+# German Electra Uncased
+<img width="300px" src="https://raw.githubusercontent.com/German-NLP-Group/german-transformer-training/master/model_cards/german-electra-logo.png">

```

Comment:
Why Palpatine? Any specific reason for this visualization? (asking out of curiosity)

Reply:
Because of "Electra" and the lightnings.